relay/config.go

```diff
@@ -1,14 +1,61 @@
 package relay
 
-import core "github.com/Layr-Labs/eigenda/core/v2"
+import (
+	"fmt"
+	"github.com/Layr-Labs/eigenda/common"
+	"github.com/Layr-Labs/eigenda/common/aws"
+	core "github.com/Layr-Labs/eigenda/core/v2"
+	"github.com/spf13/viper"
+	"os"
+	"strings"
+)
+
+const (
+	// relayEnvPrefix is the prefix for all environment variables used by the relay.
+	relayEnvPrefix = "RELAY"
+)
 
 // Config is the configuration for the relay Server.
+//
+// Environment variables are mapped into this struct by taking the name of the field in this struct,
+// converting to upper case, and prepending "RELAY_". For example, "BlobCacheSize" can be set using the
+// environment variable "RELAY_BLOBCACHESIZE".
```

Comment:
Is it a constraint of viper? This is slightly different format than what we currently use, which is underscore connected, e.g. `MAX_BLOB_SIZE`.

Reply:
There is a way, but lets leave that to a follow up PR (as discussed offline).